General/IcebergOnWindows.md

```diff
@@ -0,0 +1,43 @@
+### Long file names in Windows
```

Comment:
You should create a level-1 heading (single `#`) with the same title as the entry in the README.MD file.

Under this heading a table of contents should be added (you can use [this link](https://ecotrust-canada.github.io/markdown-toc/) to make it easier).

Second, the other titles should be level-2 heading (`##`).

Reply:
Additionally, between the level-1 heading and the toc, a short sentence describing the purpose of the page would be nice.